.buildkite/pipeline.yml

```diff
@@ -0,0 +1,35 @@
+steps:
+  - command: |
+      # set up environment
+      export PATH=/usr/local/miniconda/bin:/usr/local/bin:$$PATH
+      echo $$PATH
+      export LD_LIBRARY_PATH=$$LD_LIBRARY_PATH:/usr/local/lib
+      echo $$LD_LIBRARY_PATH
```

Comment:
IIRC, the double $$ were required to escape the $ symbol in the scripting interface.  Just in case this pops up in debugging (not sure if it's the same syntax for the yaml interface)